tools/automate.py

```diff
@@ -171,7 +171,7 @@ def setup_options(docopt_args):
         if hasattr(Options, key2) and value is not None:
             setattr(Options, key2, value)
 
-    Options.tools_dir = os.path.dirname(os.path.realpath(__file__))
+    Options.tools_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
The behaviour of realpath on Windows changed in Python 3.8. When building on a subst-ed drive, it works out the real path on the original drive and folder. This was breaking our subst-ed build_scripts build. Changing to abspath resolves the issue.